apps/cli/lib/config.ex

```diff
@@ -15,6 +15,16 @@ defmodule CLI.Config do
   """
   @spec db_name(atom()) :: nonempty_charlist()
   def db_name(chain_id) do
-    'db/mana-' ++ Atom.to_charlist(chain_id)
+    "db/mana-#{Atom.to_string(chain_id)}"
+    |> relative_to_pwd()
+    |> String.to_charlist()
+  end
+
+  @spec relative_to_pwd(String.t()) :: String.t()
+  defp relative_to_pwd(rel) do
```

Comment:
Might [Path.relative_to_cwd/1](https://hexdocs.pm/elixir/Path.html#relative_to_cwd/1) be useful here?